sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterOptions.cs

```diff
@@ -85,9 +86,34 @@ public enum ServiceVersion
         /// </summary>
         public bool DisableOfflineStorage { get; set; }
 
+        /// <summary>
+        /// Enables or disables the Live Metrics feature. This property is enabled by default.
+        /// Note: Enabling Live Metrics incurs no additional billing or costs. However, it does introduce
+        /// a performance overhead due to extra data collection, processing, and networking calls. This overhead
+        /// is only significant when the LiveMetrics portal is actively used in the UI. Once the portal is closed,
+        /// LiveMetrics reverts to a 'silent' mode with minimal to no overhead.
+        /// <see href="https://learn.microsoft.com/azure/azure-monitor/app/live-stream"/>.
+        /// </summary>
+        public bool EnableLiveMetrics { get; set; } = true;
```

Comment:
This only works with `UseAzureMonitorExporter`? Might want to note that in the remarks/summary.